lib/Doctrine/ODM/PHPCR/Tools/Helper/TranslationConverter.php

```diff
@@ -0,0 +1,256 @@
+        $class,
+        array $fields = array(),
+        $previousStrategyName = NonTranslatedStrategy::NAME,
+        &$strict = true
+    ) {
+        /** @var ClassMetadata $currentMeta */
+        $currentMeta = $this->dm->getClassMetadata($class);
+        $currentStrategyName = $currentMeta->translator ?: NonTranslatedStrategy::NAME;
+
+        // sanity check strategies
+        if ($currentStrategyName === $previousStrategyName) {
+            $message = 'Previous and current strategy are the same.';
+            if (NonTranslatedStrategy::NAME === $currentStrategyName) {
+                $message .= ' To untranslate a document, you need to specify the previous translation strategy';
+            } else {
+                $message .= sprintf(' Document is currently at %s', $currentStrategyName);
+            }
+            throw new InvalidArgumentException($message);
+        }
+
+        $translated = null;
+        foreach ($fields as $field) {
+            $current = !empty($currentMeta->mappings[$field]['translated']);
+            if (null !== $translated && $current !== $translated) {
+                throw new InvalidArgumentException(sprintf(
+                    'The list of specified fields %s contained both translated and untranslated fields. If you want to move back to untranslated, specify only the untranslated fields.',
+                    implode(', ', $fields)
+                ));
+            }
+            $translated = $current;
+        }
+        $partialUntranslate = false;
+        if (false === $translated && NonTranslatedStrategy::NAME !== $currentStrategyName) {
+            // special case, convert fields back to untranslated
+            $partialUntranslate = true;
+            $previousStrategyName = $currentStrategyName;
+            $currentStrategyName = NonTranslatedStrategy::NAME;
+            $currentMeta->translator = null;
+        }
+
+        $currentStrategy = NonTranslatedStrategy::NAME === $currentStrategyName
+            ? new NonTranslatedStrategy($this->dm)
+            : $this->dm->getTranslationStrategy($currentMeta->translator);
+
+        if (NonTranslatedStrategy::NAME === $previousStrategyName) {
+            $previousStrategy = new NonTranslatedStrategy($this->dm);
+        } else {
+            $previousStrategy = $this->dm->getTranslationStrategy($previousStrategyName);
+        }
+
+        if (!$fields) {
+            if (NonTranslatedStrategy::NAME === $currentStrategyName) {
+                throw new InvalidArgumentException('To untranslate a document, you need to specify the fields that where previously translated');
+            }
+            $fields = $currentMeta->translatableFields;
+        }
+
+        // trick query into using the previous strategy
+        $currentMeta->translator = NonTranslatedStrategy::NAME === $previousStrategyName ? null : $previousStrategyName;
+
+        // TODO do we need to change strategy on ancestors as well?
```

Comment:
ok, this is only a problem in non-strict mode. 
when we convert a child of $class, the metadata of that child will be used, not the metadata of the parent. hence we would need to change the strategy of the child class metadata. how to do that? do we register a metadata load event listener on the fly? or re-load documents with fixed metadata if they are not strictly the same class?